### ISSUE TITLE
[FR]: Recidive with journald

### ISSUE THREAD

**Issue by MystaraTheGreat** [fail2ban/fail2ban#3613]
### Environment:

- Fail2Ban version  : 1.0.2
- OS, including release name/version : Debian 11

#### Service, project or product which log or journal should be monitored

- Name of filter or jail in Fail2Ban (if already exists) : recidive
- Service, project or product name, including release name/version : recidive
- Repository or URL (if known) : 
- Service type : 
- Ports and protocols the service is listening : 

#### Log or journal information

- Journal identifier or unit name : fail2ban

#### Any additional information

It seems that recidive does not match journald entries for bans.  That is, when fail2ban is itself logging to journald, recidive does not seem to match any entries.

#### failures in sense of fail2ban filter (fail2ban must match):

```
Oct 09 23:23:50 test.com fail2ban[52973]: [sshd] Ban 218.92.0.118
```

#### legitimate messages (fail2ban should not consider as failures):

```
```

**Comment by sebres** [comment]
Recidive jail is a rudiment and retained as it is for backwards-compatibility only.
Better is to use `bantime.increment` feature instead.

However it is planned to create new backend `recidive`, which doesn't monitor any file/journal anymore, but will be directly notified about bans (= single attempt) from observer module.

So long as this is not implemented yet, you can use this jail config for that:
```ini
[recidive]
filter =
backend = systemd
failregex = ^\s*\S+ fail2ban\[\d+\]: \[[^\]]+\] Ban <ADDR>
journalmatch = _SYSTEMD_UNIT=fail2ban.service + _COMM=fail2ban
enabled = true
```

**Comment by MystaraTheGreat** [comment]
Thanks!
Yes, I had just seen the bantime.increment feature and it's probably better for my use case :)

**Comment by sebastianw** [comment]
@sebres Where can I find more information about the bantime.increment feature? I was unable to find documentation for it.

**Comment by sebres** [comment]
Fixed in #3693
